src/uu/ptx/src/ptx.rs

```diff
@@ -113,9 +125,23 @@ impl WordFilter {
         } else {
             (false, HashSet::new())
         };
-        if matches.is_present(options::BREAK_FILE) {
-            return Err(PtxError::NotImplemented("-b").into());
-        }
+        let break_set: Option<HashSet<char>> = if matches.is_present(options::BREAK_FILE)
+            && !matches.is_present(options::WORD_REGEXP)
+        {
+            let chars =
+                read_char_filter_file(matches, options::BREAK_FILE).map_err_context(String::new)?;
+            let mut hs: HashSet<char> = if config.gnu_ext {
+                HashSet::new() // really only chars found in file
```

Comment:
is it possible to have a test covering this?
thanks

Reply:
sorry not sure what you mean by "this".

side note: ptx does not support gnu-ext to be true (as of today) but I implemented the behaviour already for it for the future when somebody adds GNU extensions = testing for gnu_ext to be true in the integration test would be an error at argument parsing level. there is already other code doing so too.

Reply:
This = this line, sorry :)

Reply:
Ah, the default view doesn't show the line number just a block of code, but in the detail view it says line 134, hence my confusion. Well it is true this is not covered, because as I mentioned the GNU extensions are not implemented and if `config.gnu_ext = true` is the goal it would not pass cmd-arg parsing see line 231 - meaning a test for this would either fail or not check this line either. Line 293 by that definition is not covered too btw. If there's an issue for implementing the gnu extensions for ptx I suggest adding a test for L134 being an additional task there? Or keep that PR open until somebody does the gnu extensions? Not sure how to proceed here, sorry.